lib/algolia/insights.rb

```diff
@@ -0,0 +1,124 @@
+module Algolia
+
+  class Insights
+    def initialize(app_id, api_key, region, params = {})
+      headers = params[:headers] || {}
+      @app_id   = app_id
+      @api_key  = api_key
+      @url = "https://insights.#{region}.algolia.io"
+      @headers  = headers.merge({
+        Protocol::HEADER_APP_ID  => app_id,
+        Protocol::HEADER_API_KEY => api_key,
+        'Content-Type'           => 'application/json; charset=utf-8',
+        'User-Agent'             => ["Algolia for Ruby (#{::Algolia::VERSION})", "Ruby (#{RUBY_VERSION})"].join('; ')
+                                })
+    end
+
+    def user(user_token)
+      UserInsights.new(self, user_token)
+    end
+
+    def send_event(event)
+      puts event.inspect
+      exit(1)
```

Comment:
I believe this is test code right? It should be removed/

Reply:
🤣